int8 RIGHT = 1
int8 LEFT = 2

std_msgs/Header header
int8 right_or_left
mediapipe_gesture_recognition/Keypoint[] keypoints
